app/code/Magento/Pwa/Test/Mftf/Test/CreateTestData.xml

```diff
@@ -0,0 +1,87 @@
+           </createData>
+           <createData entity="AddToDefaultSet" stepKey="createConfigAddToAttributeSet">
+               <requiredEntity createDataKey="createConfigProductAttribute"/>
+           </createData>
+           <getData entity="ProductAttributeOptionGetter" index="1" stepKey="getConfigAttributeOption1">
+               <requiredEntity createDataKey="createConfigProductAttribute"/>
+           </getData>
+           <getData entity="ProductAttributeOptionGetter" index="2" stepKey="getConfigAttributeOption2">
+               <requiredEntity createDataKey="createConfigProductAttribute"/>
+           </getData>
+
+           <!-- Create the 2 children that will be a part of the configurable product -->
+           <createData entity="ApiSimpleOne" stepKey="createConfigChildProduct1">
+               <requiredEntity createDataKey="createConfigProductAttribute"/>
+               <requiredEntity createDataKey="getConfigAttributeOption1"/>
+           </createData>
+           <createData entity="ApiSimpleTwo" stepKey="createConfigChildProduct2">
+               <requiredEntity createDataKey="createConfigProductAttribute"/>
+               <requiredEntity createDataKey="getConfigAttributeOption2"/>
+           </createData>
+
+           <!-- Assign the two products to the configurable product -->
+           <createData entity="ConfigurableProductTwoOptions" stepKey="createConfigProductOption">
+               <requiredEntity createDataKey="createConfigProduct"/>
+               <requiredEntity createDataKey="createConfigProductAttribute"/>
+               <requiredEntity createDataKey="getConfigAttributeOption1"/>
+               <requiredEntity createDataKey="getConfigAttributeOption2"/>
+           </createData>
+           <createData entity="ConfigurableProductAddChild" stepKey="createConfigProductAddChild1">
+               <requiredEntity createDataKey="createConfigProduct"/>
+               <requiredEntity createDataKey="createConfigChildProduct1"/>
+           </createData>
+           <createData entity="ConfigurableProductAddChild" stepKey="createConfigProductAddChild2">
+               <requiredEntity createDataKey="createConfigProduct"/>
+               <requiredEntity createDataKey="createConfigChildProduct2"/>
+           </createData>
+       </before>
+
+        <after>
+            <!--Delete created entity -->
+            <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
+            <deleteData createDataKey="createConfigProduct" stepKey="deleteConfigProduct"/>
+            <deleteData createDataKey="createConfigChildProduct1" stepKey="deleteConfigChildProduct1"/>
+            <deleteData createDataKey="createConfigChildProduct2" stepKey="deleteConfigChildProduct2"/>
+            <deleteData createDataKey="createConfigProductAttribute" stepKey="deleteConfigProductAttribute"/>
+        </after>
+    </test>
+
+
+
+</tests>
```

Comment:
Remove extra whitespace above and add line break

Reply:
Done